### DISCUSSION TITLE
posts/review/2024-Adieu

### DISCUSSION THREAD

**Discussion by giscus** [chhw130/mincho-log#27] [Post Comment]
# posts/review/2024-Adieu

2024 참 많은 일과 활동을 했었다. 개인적으로 올해 시간을 참 알차게 썼다는 생각이 든다. 그만큼 시간을 낭비하고 살지는 않았던 거 같다. 회고를 작성하면서 한 번 더 돌이켜보고 싶었다.

https://www.mincho130.xyz/posts/review/2024-Adieu

**Comment by Seojunhwan** [comment]
현우님~! 너무 잘 읽었어요! 특히 주도적으로 개선해주신 모습이 너무 감명 깊었습니다! 24년도에 현우님을 뵙게되어 너무 좋았습니다!

**Reply by chhw130** [reply]
@Seojunhwan 오 준환님~ 오랜만입니다! 저도 준환님을 알게되어서 좋았습니다! 올해도 한번 더 뵙고 싶은 분들 중 한명입니다!
읽어주셔서 감사합니다 ㅋㅋㅋ
